docs/configure-archive-logs.md

```diff
@@ -59,3 +61,25 @@ spec:
     archiveLocation:
       archiveLogs: true
 ```
+
+## Suggested alternatives
+
+Argo Workflows log storage facilities are quite basic.
+
+We recommend using a fully featured Kubernetes aware logging facility which will provide you with facilities for indexing, searching and managing of log storage.
```

Comment:
So this is repetitive with line 77 below, which is why I [previously suggested](https://github.com/argoproj/argo-workflows/pull/12597#discussion_r1473487646) that to replace this. I still think it should be re-ordered to list the alternatives first, as that is the heading of this section.

This also still has the issues mentioned in my previous comment (e.g. "facility which will provide you facilities")

Reply:
Moved and reworded.

Reply:
repetition followed up in [below comment](https://github.com/argoproj/argo-workflows/pull/12597#discussion_r1486746801)